umake/frameworks/baseinstaller.py

```diff
@@ -137,6 +136,9 @@ def remove(self):
     def confirm_path(self, path_dir=""):
         """Confirm path dir"""
 
+        if self.desktop_filename:
```

Comment:
I moved this from the end of confirm_path to the start.
The medium baseInstaller test was failing.
The function hit a return before it could set the exec_path